python/llm/src/ipex_llm/transformers/npu_pipeline_model/common.py

```diff
@@ -35,6 +36,8 @@ def update_names_of_IR_and_export_blob(model, model_name, dir, compile_blob=True
     core.set_property("NPU", {"NPU_COMPILATION_MODE_PARAMS":
                               "compute-layers-with-higher-precision=Sqrt,Power,ReduceMean,Add"})
     core.set_property("NPU", {"PERFORMANCE_HINT": "LATENCY"})
+    if npu_dpu_groups is not None:
```

Comment:
check `IPEX_LLM_NPU_DISABLE_COMPILE_OPT` (see https://github.com/intel-analytics/ipex-llm/blob/main/python/llm/src/ipex_llm/transformers/npu_models/qwen2_mp.py#L238)

Reply:
Add it.